### PR TITLE
@xtina-starr => New Input

### PR DESCRIPTION
**Pull request by l2succes**
Depends on the Type system consolidation PR (https://github.com/artsy/reaction/pull/629)

PS: the gray border is not visible in the GIF but it's there

### CODE REVIEW

File: src/Components/__stories__/Input.story.tsx [hunk 1]
```diff
@@ -8,38 +10,70 @@ import Icon from "../Icon"
...
+      <Title>Input</Title>
+      <Subtitle>Our default input style. Title is optional.</Subtitle>
+
+      <section style={{ padding: 10 }}>
+        <Input placeholder="Placeholder" title="Title" block />
+
+        <Input placeholder="Placeholder" title="Title" value="Content" block />
 
-      <div style={{ paddingTop: 10 }}>
+        <Input title="Title" placeholder="Placeholder" block disabled />
```
Comment (erikdstock) [inline src/Components/__stories__/Input.story.tsx]: Why disabled?

File: src/Components/__stories__/Input.story.tsx [hunk 2]
```diff
@@ -8,38 +10,70 @@ import Icon from "../Icon"
...
+    <div style={{ padding: 10 }}>
+      <Title>Input with label</Title>
+      <Subtitle>Used for short/simple forms</Subtitle>
+
+      <div style={{ padding: 10 }}>
+        <Input
+          placeholder="Enter your email address"
+          label="Email"
+          block
+          quick
```
Comment (erikdstock) [inline src/Components/__stories__/Input.story.tsx]: `quick` means the label is inside the Input, but there are also inputs with external labels, right?

Comment (erikdstock) [reply src/Components/__stories__/Input.story.tsx]: Ah, I was thinking that was part of the input. Thanks.

**Comment by l2succes** [comment]
@erikdstock @xtina-starr It's finally ready if you guys can take another look. Thanks :)

**Pull request merged**